src/components/Section/Identification/ContactInformation/ContactInformation.jsx

```diff
@@ -15,6 +15,7 @@ import { IDENTIFICATION, IDENTIFICATION_CONTACTS } from 'config/formSections/ide
 
 import connectSubsection from 'components/Section/shared/SubsectionConnector'
 import Subsection from 'components/Section/shared/Subsection'
+import Generic from '../../../Form/Generic'
```

Comment:
It doesn't look like this import is used

Reply:
Pushed a fix to remove this from the code.